client/src/dataset/components/DatasetOpen.tsx

```diff
@@ -50,6 +51,14 @@ const getDefaultDSType = (didReset: boolean, openState: OpenDatasetState) => {
  * @param openState complete OpenDatasetState instance
  */
 
+ // Fix this after separating info from params
+ const mergeCachedParamsInfo = (formCachedParams: any, formDetectedParams: any) => {
+     Object.keys(formDetectedParams).filter((key: string) => isAdditionalInfo(key)).forEach((key: string) => {
+       formCachedParams[key] = formDetectedParams[key];
+     });
```

Comment:
`.filter((key: string) => isAdditionalInfo(key))` can be replaced with just `.filter(isAdditionalInfo)`, as `isAdditionalInfo` already matches the signature of functions that are expected by `Array.filter`.

Instead of modifying `formCachedParams`, which can be confusing to the calling function, it would be preferable to return a new `Array`. Maybe use `Array.map` here instead?